File: fifa2014group.py
import sys
import requests

group_name=sys.argv[1]
if (group_name > "H" or group_name < "A"):
	print ('Need just one commandline argument, A...H')
else:
	teams = requests.get("http://worldcup.kimonolabs.com/api/teams?sort=groupRank&apikey=MvtW5flSePbaulwoJCbeExYvIpvnQexL")
	teams = teams.json()
	print "{0:<18} {1:^2} {2:^2} {3:^2}".format("Name","W", "D", "L")
	for item in teams:
		if item['group'] == group_name:
			name = item['name'].encode("utf-8")
			print "{0:<18} {1:^2} {2:^2} {3:^2}".format(name, item['wins'], item['draws'], item['losses'])

File: homework2/fifa2014group.js
var request = require ('request');
var sprintf = require ("sprintf-js").sprintf;


if (process.argv.length !== 3 || !/^[A-H]$/.test(process.argv[2])) {
	console.error('Need just one command line argument, A...H');
	process.exit(1);
}

var api = 'http://worldcup.kimonolabs.com/api/teams?group=' + process.argv[2] + '&sort=groupRank&apikey=MvtW5flSePbaulwoJCbeExYvIpvnQexL' ;

request(api, function (error, response, body) {
	if (!error && response.statusCode == 200) 
	console.log(sprintf("%-19s%3s%3s%3s", 'Name', 'W', 'D', 'L'));
	JSON.parse(body).forEach(function (team) {
		console.log(sprintf("%-19s%3d%3d%3d", team.name, team.wins, team.draws, team.losses));
	} )
})

//https://www.npmjs.org/package/request
